jupyterlab/Dockerfile.conda

```diff
@@ -0,0 +1,33 @@
+FROM continuumio/anaconda3
+
+WORKDIR /root
+
+RUN apt-get update
+
+# Install java
+# ------------
+RUN apt-get install -y openjdk-8-jdk
+RUN apt-get install ca-certificates-java && update-ca-certificates -f
```

Comment:
Merge L9 + L10, ensure apt is launched in non interactive mode.

Reply:
The merge of L9 and L10 gives some unexpected errors.